test/validateCardNumber_spec.coffee

```diff
@@ -43,10 +43,6 @@ describe 'payform', ->
       assert(payform.validateCardNumber('4917300800000000'), 'visaelectron')
 
       assert(payform.validateCardNumber('6759649826438453'), 'maestro')
-      assert(payform.validateCardNumber('639002000000000003'), 'maestro')
```

Comment:
You can keep those 4 tests, they are from this commit: https://github.com/jondavidjohn/payform/commit/5257fca97a0f05cbb3ef78f7e406749fbd6d6fc9